package.json

```diff
@@ -46,7 +46,7 @@
     "eslint-plugin-import": "^2.14.0",
     "eslint-plugin-jsx-a11y": "^6.1.2",
     "eslint-plugin-react": "^7.11.1",
-    "jest-junit": "^5.2.0",
+    "jest-junit": "^4.0.0",
```

Comment:
What's the reason for the downgrade?

Reply:
The newer version causes an error saying I should configure a reporter in jest when I run `npm run test:bamboo`. But configuring a new reporter only conflicts with what jest uses when the app was created using Create React App. So I had to downgrade to the one that was previously there

Reply:
Awesome! I get it now.